types/workbox-build/workbox-build-tests.ts

```diff
@@ -0,0 +1,73 @@
+/* tslint:disable:comment-format no-namespace */
+
+'use strict';
+
+import {
+    copyWorkboxLibraries,
+    injectManifest,
+    InjectManifestConfig,
+    InjectManifestResult,
+    generateSW,
+    GenerateSWConfig,
+    GenerateSWResult,
+    getManifest,
+    GetManifestConfig,
+    GetManifestResult,
+    getModuleURL,
+} from 'workbox-build';
+
+//==============================================================================
+// WorkboxBuild.copyWorkboxLibraries
+//==============================================================================
+
+export namespace CopyWorkboxLibrariesTest {
```

Comment:
Why does the test export a namespace?

Reply:
To be honest, I'm not sure. I was copying the style in the other [workbox] types, e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/workbox-core/workbox-core-tests.ts. I can remove if this is an issue